### DISCUSSION TITLE
能否添加为视频点踩的选项?

### DISCUSSION THREAD

**Discussion by lolisix** [the1812/Bilibili-Evolved#1903] [综合讨论 / General]
点踩后对视频有没有用先不提,主要是有些视频制作的就离谱,不点踩我浑身难受.

**Comment by Geeyun-JY3** [comment]
视频离不离谱、你难不难受先不提，主要是 Bilibili Evolved 也不是全能神。

还有就是重复：#342、#850
下次提需求前请先善用搜索。

**Reply by lolisix** [reply]
搜索不仔细,怪我了,怪我了!
